### PR TITLE
documentation and version number bump

### PR DESCRIPTION
**Pull request by lziq**
Issue #, if available:

### Description of changes:
* Changed version number. 
* Changed README file.

### CODE REVIEW

File: client/package-lock.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "ion-extension-client",
-	"version": "0.0.1",
+	"version": "0.2.0",
```
Comment (therapon) [inline client/package-lock.json:3]: Just checking here, you did not manually modify the `package-lock.json`, right? It is automatically managed, see https://docs.npmjs.com/configuring-npm/package-lock-json.html

File: README.md [hunk 2]
```diff
@@ -8,6 +8,8 @@ VSCode extension (plugin) that supports the Ion data format.
 ## Functionality 
 
 * Syntax highlighting. 
+* Error reporting. 
+* Formatting.
```
Comment (therapon) [inline README.md:12]: We should add a sentence to each item that this happens on the whole file and not while typing for now.

File: README.md [hunk 3]
```diff
@@ -8,6 +8,8 @@ VSCode extension (plugin) that supports the Ion data format.
 ## Functionality 
 
 * Syntax highlighting. 
+* Error reporting. 
+* Formatting. 
 
 ## Development
```
Comment (therapon) [inline README.md:15]: We should also add the instructions on how to check out the repo. You can reuse the text and from one of our other libs here.

File: client/package-lock.json (hunk 1 above)
Comment (lziq) [reply client/package-lock.json:3]: I did manually modify it.

Comment (lziq) [reply client/package-lock.json:3]: I will remove this change.

Comment (therapon) [reply client/package-lock.json:3]: Read the link I pasted in my comment. Letting npm manage package-lock.json and then adding it to the repo is what we should do here.

File: README.md [hunk 4]
```diff
@@ -3,14 +3,18 @@
 
 # VSCode extension for Ion 
 
-VSCode extension (plugin) that supports the Ion data format.  
+VSCode extension (plugin) that supports the Ion data format. You can download this extension from VSCode marketplace.
```
Comment (therapon) [inline README.md:6]:
```suggestion
[VSCode extension (plugin)](https://marketplace.visualstudio.com/items?itemName=TheIonTeam.ion-extension) that supports the Ion data format.
```

File: README.md [hunk 5]
```diff
@@ -3,14 +3,18 @@
 
 # VSCode extension for Ion 
 
-VSCode extension (plugin) that supports the Ion data format.  
+VSCode extension (plugin) that supports the Ion data format. You can download this extension from VSCode marketplace. 
+GitHub repository: https://github.com/amzn/ion-vscode-plugin
```
Comment (therapon) [inline README.md:7]: This is a link to the page they are already on :). Remove please.

File: README.md [hunk 6]
```diff
@@ -3,14 +3,18 @@
...
+GitHub repository: https://github.com/amzn/ion-vscode-plugin 
 
 ## Functionality 
 
-* Syntax highlighting. 
-* Error reporting. 
-* Formatting. 
-
+- **Syntax highlighting** 
+    Colorize the text document.
```
Comment (therapon) [inline README.md:12]:
```suggestion
    Colorize the Ion values in your document. 
```

File: README.md [hunk 7]
```diff
@@ -3,14 +3,18 @@
...
 ## Functionality 
 
-* Syntax highlighting. 
-* Error reporting. 
-* Formatting. 
-
+- **Syntax highlighting** 
+    Colorize the text document. 
+-  **Error reporting** 
+    Give a Squiggly line for the error part. For now, it only finds the first error in the editor.
```
Comment (therapon) [inline README.md:14]:
```suggestion
    Reports errors in the editor and the "Problems" tab. Currently supports whole document validation. 
```

File: README.md [hunk 8]
```diff
@@ -3,14 +3,18 @@
...
-* Syntax highlighting. 
-* Error reporting. 
-* Formatting. 
-
+- **Syntax highlighting** 
+    Colorize the text document. 
+-  **Error reporting** 
+    Give a Squiggly line for the error part. For now, it only finds the first error in the editor. 
+- **Formatting** 
+    Press `Shift + Option + F`, then type `Format Document`.
```
Comment (therapon) [inline README.md:16]:
```suggestion
    Formats the whole document. Currently support whole error-free document formatting. 
```

File: client/package-lock.json [hunk 9]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "ion-extension-client",
-	"version": "0.2.0",
+	"version": "0.0.1",
```
Comment (therapon) [inline client/package-lock.json]: See https://stackoverflow.com/questions/46653833/is-there-a-way-to-force-npm-to-generate-package-lock-json for how to force an update on package-lock.json

Comment (lziq) [reply client/package-lock.json]: Sure. I will use `npm install --package-lock` to force the update for package-lock.json file.

File: README.md [hunk 10]
```diff
@@ -3,17 +3,17 @@
...
 
 - **Syntax highlighting** 
-    Colorize the text document. 
+    Colorize the Ion values in your document. 
 -  **Error reporting** 
-    Give a Squiggly line for the error part. For now, it only finds the first error in the editor. 
+    Reports errors in the editor and the "Problems" tab. Currently supports whole document validation. 
 - **Formatting** 
-    Press `Shift + Option + F`, then type `Format Document`. 
+    Formats the whole document. Currently support whole error-free document formatting.
```
Comment (therapon) [inline README.md:15]:
```suggestion
    Formats the whole document. Currently supports whole error-free document formatting. 
```

File: README.md [hunk 11]
```diff
@@ -3,17 +3,17 @@
...
 - **Syntax highlighting** 
-    Colorize the text document. 
+    Colorize the Ion values in your document. 
 -  **Error reporting** 
-    Give a Squiggly line for the error part. For now, it only finds the first error in the editor. 
+    Reports errors in the editor and the "Problems" tab. Currently supports whole document validation. 
 - **Formatting** 
-    Press `Shift + Option + F`, then type `Format Document`. 
+    Formats the whole document. Currently support whole error-free document formatting. 
+Usage: Press `Shift + Option + F`, then select `Format Document`.
```
Comment (therapon) [inline README.md:16]: This is VSCode standard, it is also available through the VScode menu I think we can remove this line

**Pull request merged**